### PR TITLE
proposal for the ios readme file

### PR DESCRIPTION
**Pull request by CyrusZei**
This is just a proposal for the readme file

### CODE REVIEW

File: docs/v2/ios.md [hunk 1]
```diff
@@ -1,53 +1,149 @@
...
 ```
 
-# Step 1. install pod dependencies
-```bash
-cd ios
-pod install
+#### 6. we need to initialize pod
+if you don't have cocopods installed run this command in your terminal
+```sh
+$ sudo gem install Cocoapods
```
Comment (Matrixbirds) [inline docs/v2/ios.md]: I don't think you probably need the sudo here. U should grant the current user gem install permission.

Comment (CyrusZei) [reply docs/v2/ios.md]: Totally agree about that, let me change that

Comment (CyrusZei) [reply docs/v2/ios.md]: Removed the `sudo` command

File: docs/v2/ios.md [hunk 2]
```diff
@@ -1,53 +1,149 @@
-# iOS Installation
+# React native Agora iOS Installation
+#### Minimum compatibility
+* React native 0.58.x
+* IOS SDK 8.0+
```
Comment (Matrixbirds) [inline docs/v2/ios.md:4]: iOS should be more general

File: docs/v2/ios.md [hunk 3]
```diff
@@ -1,53 +1,149 @@
...
+* Android 5.0+ x86 arm64 armv7a
 
-# Prerequisites
-* You should installed and linked `react-native-agora`
-* Xcode 10.0+
-* cocoapods
+#### 1. open up your terminal and write this :
+When it comes to the version nr you can put any version you like as long as it it compatibility
```
Comment (Matrixbirds) [inline docs/v2/ios.md:8]: I think abbreviation is only comfortable for  native english speaker. probably u need the unfolding the `nr`

**Pull request merged**